client/src/routes/login/Login.tsx

```diff
@@ -1,30 +1,205 @@
+import { Box, Button, CardActions, CardContent, Checkbox, FormControlLabel, FormGroup, Input, Link, Paper, Tab, Tabs } from '@mui/material';
 import Card from '@mui/material/Card';
 import CardActionArea from '@mui/material/CardActionArea';
+import TextField from '@mui/material/TextField';
 import Typography from '@mui/material/Typography';
 import {
 	Container
 } from '@mui/system';
+import { DatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import dayjs, { Dayjs } from 'dayjs';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import React from 'react';
+import { Form } from 'react-router-dom';
+
+interface TabPanelProps {
+	children?: React.ReactNode;
+	index: number;
+	value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+	const { children, value, index, ...other } = props;
+
+	return (
+		<Box
+			role="tabpanel"
+			hidden={value !== index}
+			id={`simple-tabpanel-${index}`}
+			aria-labelledby={`simple-tab-${index}`}
+			{...other}
+		>
+			{value === index && (
+				<Box sx={{ p: 3 }}>
+					{children}
+				</Box>
+			)}
+		</Box>
+	);
+}
+
+function a11yProps(index: number) {
+	return {
+		id: `simple-tab-${index}`,
+		'aria-controls': `simple-tabpanel-${index}`,
+	};
+}
 
 function Login() {
+
+	const signIn = (e: React.FormEvent<HTMLFormElement>) => {
+		e.preventDefault();
+		
+		// logs voor testen
+		console.log(email);
+		console.log(password);
+		console.log(rememberMe);
+	}
+	
+	const signUp = (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
gebruik een useCallback(()=>,[]) voor de functie. memoizes the method so it doen't have to redo the method.